### PR TITLE
fix footer spacing css so that footer is more responsive

### PR DESCRIPTION
**Pull request by clairenied**
This PR addresses this issue, here:
https://github.com/openaq/openaq.org/issues/972

You can see that the signup form was originally breaking out of the footer, causing the form fields to no longer be visible. In this PR, I make the footer a little more responsive to different screen sizes, so this should be less of an issue.

I tested this locally, please let me know if it looks good from your end, as well! Happy to make any additional changes you require.

### CODE REVIEW

File: themes/openaq/assets/scss/_footer.scss [hunk 1]
```diff
@@ -1,30 +1,30 @@
-@use 'openaq-design-system/variables';
+@use "openaq-design-system/variables";
 
 .footer {
-  height: 250px;
```
Comment (clairenied) [inline themes/openaq/assets/scss/_footer.scss:4]: Remove hard-coded width and height so form input fields can wrap

File: themes/openaq/assets/scss/_footer.scss [hunk 2]
```diff
@@ -63,30 +72,45 @@
...
+
         .copyright {
           order: 1;
         }
       }
     }
   }
 
-  @media (max-width: 1000px) {
+  @media (max-width: 1180px) {
```
Comment (clairenied) [inline themes/openaq/assets/scss/_footer.scss:83]: Adjust media query to reflect new standardized form field sizes

File: themes/openaq/assets/scss/_footer.scss [hunk 3]
```diff
@@ -63,30 +72,45 @@
...
     & {
       &__content {
-        padding: 0 15px;
+        .nav-section {
```
Comment (clairenied) [inline themes/openaq/assets/scss/_footer.scss:86]: Add justification now that form moves more with responsive features

File: themes/openaq/assets/scss/_footer.scss [hunk 4]
```diff
@@ -105,15 +129,27 @@
...
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+
+            input {
+              width: 300px;
+            }
+          }
+
+          .social-links {
```
Comment (clairenied) [inline themes/openaq/assets/scss/_footer.scss:148]: Make sure social links don't also break out of the bounding box

File: themes/openaq/layouts/partials/footer.html [hunk 5]
```diff
@@ -3,9 +3,7 @@
     <section class="nav-section">
```
Comment (clairenied) [inline themes/openaq/layouts/partials/footer.html:3]: Any stylistic changes in here just automatically come from my prettier config, I am unopinionated so just let me know if you want me to remove 😄

File: yarn.lock
Comment (clairenied) [inline yarn.lock:1]: This got regenerated when I set up the repo, hope that's ok.

**Comment by clairenied** [comment]
@russbiggs Here we go!

**Comment by russbiggs** [comment]
This looks great, thanks for the fix!

**Pull request merged**